.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -5,9 +5,17 @@ labels: 'bug'
 title: "[BUG]"
 ---
 Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)
-for information on what types of issues we address. For help with debugging your code, please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
+for additional information about bug reports. For help with debugging your code, please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
 
-Please fill in this template and do not delete it unless you are sure your issue is outside its scope.
+**Please fill in this bug report template to ensure a thorough response.**
```

Comment:
Do I want a _thorough_ response? Quick? Some other words?

Reply:
I've changed this to `timely and thorough`. More than happy to iterate here though!

Reply:
^ I applied these changes to the PR template as well